.github/workflows/rollup-config-diff.yml

```
@@ -0,0 +1,39 @@
+name: Rollup Config Diff
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  compute-rollup-config-diff:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

v4 is currently the latest
```suggestion
      uses: actions/checkout@v4
```